### PR TITLE
Fix residual histogram scaling

### PR DESCRIPTION
**Pull request by koenderks**

### CODE REVIEW

File: JASP-Engine/JASP/R/regressionlinear.R [hunk 1]
```diff
@@ -3938,11 +3938,11 @@ RegressionLinear <- function(dataset=NULL, options, perform="run", callback=func
 
     h <- hist(res, plot = FALSE)
     dens <- h$density
-    yhigh <- 1.2*max(h$density)
```
Comment (TimKDJ) [inline JASP-Engine/JASP/R/regressionlinear.R]: `dens` is not a list, but a vector; it should be `max(c(h$density, dens))`

File: JASP-Engine/JASP/R/regressionlinear.R [hunk 2]
```diff
@@ -3938,11 +3938,11 @@ RegressionLinear <- function(dataset=NULL, options, perform="run", callback=func
...
+		yhigh <- max(c(h$density, dens$y))
```
Comment (TimKDJ) [inline JASP-Engine/JASP/R/regressionlinear.R]: `dens` is not a list, but a vector; it should be `max(c(h$density, dens))`

**Comment by koenderks** [comment]
Fixed, but not as suggested in Tim's comment. The yhigh should be the maximum of the density line and the density of the histogram. Now this is implemented correctly.

**Comment by AlexanderLyNL** [comment]
Can you fix the unit test?

**Comment by koenderks** [comment]
> Can you fix the unit test?

The unit test is about the plot right? Is that not supposed to be?

**Comment by AlexanderLyNL** [comment]
Update the svg?

**Comment by koenderks** [comment]
> Update the svg?

Can you point me to where to do this?

**Comment by AlexanderLyNL** [comment]
The test fails because the new svg looks different from the reference: 

/jasp-desktop/JASP-Tests/R/tests/figs/RegressionLinear/regressionlinear-residuals-histogram.svg

In my folder, this reference looks quite okay. What does the new one look like?

**Comment by koenderks** [comment]
I'm not sure what data is used in the unit test to make that svg

**Comment by AlexanderLyNL** [comment]
Let's talk about this tomorrow, perhaps @TimKDJ can help.

**Comment by koenderks** [comment]
Tuesday will have to do, as I have to teach on Monday and Wednesday.

**Comment by AlexanderLyNL** [comment]
@koenderks Could you also have "Standardized residuals" on by default?

**Comment by TimKDJ** [comment]
Can you rebase? The test should be ok afterwards

**Comment by koenderks** [comment]
Rebased

**Pull request merged**